Yoast/javascript
# QAK-2247 Fix the Arabic passive voice assessment bug

Also add Arabic paper to test

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds word length limitation to the function for detecting Arabic passive voice and add Arabic paper for testing.
* [Yoast SEO Free] Fixes a bug in which a sentence containing single-letter word would trigger `An error occurred in the 'passiveVoice' assessment` in passive voice assessment bullet.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Set the WP environment language to Arabic
* Test this functionality in Wordpress SEO Free
* You should see a bullet assessment for passive verb forms in the Readability section
* Add these sentences to your post:
`بدلاً من رميها، يمكنك زرع فص الثوم الذي ينبت على عمق حوالي 1 بوصة في وعاء 4 بوصات وسقيها.
حماسي في تلك الأيام أثير كلياً بواسطة الكحول.`
`101 نشاطًا رائعًا للأطفال من سن 5 إلى 8 سنوات.`
`أضف ٢ إلى ٣ ملاعق من العسل.`
* Check that the passive voice bullet is not grey and does not give `An error occurred in the 'passiveVoice' assessment` message.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #https://yoast.atlassian.net/browse/QAK-2247
